### PR TITLE
Simplify rules ignore in light of `v0.1.3`

### PR DESCRIPTION
**Pull request by paddyroddy**
I noticed in the release notes that they've toned down recommendations for which clash https://github.com/astral-sh/ruff/releases/tag/v0.1.3

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -8,20 +8,12 @@ testpaths = [
...
-    'COM812', # ruff-format recommended
-    'COM819', # ruff-format recommended
-    'D206', # ruff-format recommended
-    'E501', # ruff-format recommended
-    'ISC001', # ruff-format recommended
-    'Q000', # ruff-format recommended
-    'Q001', # ruff-format recommended
-    'Q002', # ruff-format recommended
-    'Q003', # ruff-format recommended
-    'W191', # ruff-format recommended
```
Comment (samcunliffe) [inline pyproject.toml:24]: ✨

File: pyproject.toml [hunk 2]
```diff
@@ -8,20 +8,12 @@ testpaths = [
 fix = true
 force-exclude = true
 ignore = [
+    "COM812", # ruff-format recommended
```
Comment (samcunliffe) [inline pyproject.toml]:
```suggestion
    "COM812", # trailing commas (ruff-format recommended)
```

File: pyproject.toml [hunk 3]
```diff
@@ -8,20 +8,12 @@ testpaths = [
...
-    'COM819', # ruff-format recommended
-    'D206', # ruff-format recommended
-    'E501', # ruff-format recommended
-    'ISC001', # ruff-format recommended
-    'Q000', # ruff-format recommended
-    'Q001', # ruff-format recommended
-    'Q002', # ruff-format recommended
-    'Q003', # ruff-format recommended
-    'W191', # ruff-format recommended
+    "ISC001", # ruff-format recommended
```
Comment (samcunliffe) [inline pyproject.toml]:
```suggestion
    "ISC001", # simplify implicit str concatenation (ruff-format recommended)
```

File: {{cookiecutter.project_slug}}/pyproject.toml [hunk 4]
```diff
@@ -70,9 +70,12 @@ testpaths = [
 fix = true
 force-exclude = true
 ignore = [
+    "COM812", # ruff-format recommended
     "D203", # no-blank-line-before-class
     "D212", # multi-line-summary-first-line
+    "D407", # removed dashes lines under sections
     "D417", # argument description in docstring (unreliable)
+    "ISC001", # ruff-format recommended
```
Comment (samcunliffe) [inline {{cookiecutter.project_slug}}/pyproject.toml]:
```suggestion
    "COM812", #  trailing commas (ruff-format recommended)
    "D203", # no-blank-line-before-class
    "D212", # multi-line-summary-first-line
    "D407", # removed dashes lines under sections
    "D417", # argument description in docstring (unreliable)
    "ISC001", # simplify implicit str concatenation (ruff-format recommended)
```

File: hooks/post_gen_project.py [hunk 5]
```diff
@@ -76,7 +76,7 @@ def main(initialise_git_repository: str) -> int:
                 "https://docs.github.com/en/get-started/quickstart/create-a-repo.\n\n"
                 "Then run:\n\n"
                 "git remote add origin git@github.com:"
-                "{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}.git"
+                "{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}.git",
```
Comment (samcunliffe) [inline hooks/post_gen_project.py]:
```suggestion
                "{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}.git"
```
I'm confused, we're suppressing [COM812](https://docs.astral.sh/ruff/rules/missing-trailing-comma/) but it looks like it's been applied here.

**Review by samcunliffe — APPROVED** [review]
Some non-blocker queries.

File: hooks/post_gen_project.py (hunk 5 above)
Comment (paddyroddy) [reply hooks/post_gen_project.py]: Hmm, if linting is passing then might be a bug?

Comment (paddyroddy) [reply hooks/post_gen_project.py]: Or I ran it first, before ignoring the rule. Will remove

Comment (samcunliffe) [reply hooks/post_gen_project.py]: Our bikeshed is nice and waterproof now.

**Pull request merged**